web/core/layouts/auth-layout/workspace-wrapper.tsx

```diff
@@ -150,7 +150,7 @@ export const WorkspaceAuthWrapper: FC<IWorkspaceAuthWrapper> = observer((props)
         <div className="container relative mx-auto flex h-full w-full flex-col overflow-hidden overflow-y-auto px-5 py-14 md:px-0">
           <div className="relative flex flex-shrink-0 items-center justify-between gap-4">
             <div className="z-10 flex-shrink-0 bg-custom-background-90 py-4">
-              <Image src={planeLogo} className="h-[26px] w-full" alt="Plane logo" />
+              <Image src={planeLogo} height={26} className="h-[26px]" alt="Plane logo" />
```

Comment:
If this is causing an issue in Firefox, we'll need to update it across the platform. This affects around 10-15 components at least. could you check all of them, please? @anmolsinghbhatia